inc/dashboard/widget.class.php

```diff
@@ -878,20 +878,21 @@ private static function getBarsGraph(
       array $series = []): string {
 
       $defaults = [
-         'label'        => '',
-         'alt'          => '',
-         'color'        => '',
-         'icon'         => '',
-         'legend'       => false,
-         'multiple'     => false,
-         'stacked'      => false,
-         'horizontal'   => false,
-         'distributed'  => false,
-         'use_gradient' => false,
-         'point_labels' => false,
-         'limit'        => 99999,
-         'filters'      => [],
-         'rand'         => mt_rand(),
+         'label'             => '',
+         'alt'               => '',
+         'color'             => '',
+         'icon'              => '',
+         'legend'            => false,
+         'multiple'          => false,
+         'stacked'           => false,
+         'horizontal'        => false,
+         'distributed'       => false,
+         'use_gradient'      => false,
+         'point_labels'      => false,
+         'limit'             => 99999,
+         'filters'           => [],
+         'rand'              => mt_rand(),
+         'truncate_boundary' => 'beginning',
```

Comment:
new option : the 2 new dashboard cards rely on ordering to show the most important data. However, the code truncates the data on the wrong bound (because it was designed to work on data ordered by date / time).